### ISSUE TITLE
Attempt to access invalid address

### ISSUE THREAD

**Issue by Extrudejsp** [kolton/d2bot-with-kolbot#164]
Followed the guide and a few youtube videos. Seem to unable to get this working. It launched d2 in the background in my task manager but i do not see an active window.

Running windows 10 64 bit.

From what i can see i do not have to do a version hack and i prefer not to. Anyone figure out how to solve this issue?

**Comment by HansGerber** [comment]
Have you tried to run it as admin?
Have you tried to run it in compatibility mode?
What does your config file look like?

**Comment by Extrudejsp** [comment]
Yes, yes and great. This issue is not caused by my config file.

**Comment by aeloros** [comment]
I'm also seeing this issue on Win10 64-bit.

I followed a number of youtube setup guides.  It's running as Admin.  I tried a number of compatibility modes.

I'm not sure which config file you want, so I'm assuming you want the profile.ini file.  Here is what mine looks like.

`{"Account":"myaccount","Password":"mypassword","Character":"foodog","GameName":"foodog","GamePass":"foodog","D2Path":"E:\\Games\\Diablo II\\Game.exe","Realm":"East","Mode":"Battle.net","Difficulty":"Normal","Parameters":"-w","Entry":"C:\\Users\\Scott\\Desktop\\d2bot-with-kolbot-master\\d2bs\\kolbot\\D2BotLead.dbj","Location":"x, y","KeyList":"","Schedule":"","GameCount":0,"Runs":0,"Chickens":0,"Deaths":0,"Crashes":0,"Restarts":0,"RunsPerKey":0,"KeyRuns":0,"MaxRam":0,"Visible":false,"BlockRD":false,"ScheduleEnable":false,"Type":0,"Name":"main","Group":"default"}`
